src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -107,23 +107,46 @@ method java.lang.Object equals java.lang.Object
 method java.lang.Object getClass
 method java.lang.Object hashCode
 method java.lang.Object toString
+new java.lang.String
+staticField java.lang.String CASE_INSENSITIVE_ORDER
+new java.lang.String byte[]
+new java.lang.String byte[] int int
+new java.lang.String byte[] int int java.lang.String
+new java.lang.String byte[] int int java.nio.charset.Charset
+new java.lang.String byte[] java.lang.String
+new java.lang.String byte[] java.nio.charset.Charset
```

Comment:
I left this out, because `Charset` isn't otherwise allowed.

Reply:
Yeah good point, I can add the fields in `StandardCharsets`

Reply:
I added the `StandardCharsets` fields and `Charset.forName` in https://github.com/jenkinsci/script-security-plugin/pull/310/commits/ce20e20786fe950959827fddf3b53234bca302f4.